src/main/java/org/myrobotlab/service/Gpt3.java

```diff
@@ -99,12 +109,28 @@ public Response getResponse(String text) {
         @SuppressWarnings({ "unchecked", "rawtypes" })
         Map<String, Object> payload = (Map) CodecUtils.fromJson(msg, LinkedHashMap.class);

```

Comment:
This can be done without casting using StaticType. Like so:
```java
Map<String, Object> payload = CodecUtils.fromJson(msg, new StaticType{}());
```

But, a better way would be to create a POJO that maps directly to the response format, eliminating needs for later casts as well

Reply:
initially, I was excited to see they have an openapi.yml
but I tried a myriad of different versions of open-api generator and multiple versions of the spec release 2.0.0 and master
nothing worked without lots of errors when it generated the spec 
![image](https://github.com/MyRobotLab/myrobotlab/assets/1266144/1d37a6fc-4812-41ad-b97b-8f69cd8a98e8)

Reply:
you mean 
```import org.myrobotlab.framework.StaticType;```
does not seem to work ...
![image](https://github.com/MyRobotLab/myrobotlab/assets/1266144/20ff9415-e0ab-4134-a934-b2baf0dd74e0)

Reply:
Needs to be `new StaticType<>() {}`

Also, you need to remove the `(Map)` cast